Branch-SDK/src/io/branch/referral/InstallListener.java

```diff
@@ -4,23 +4,33 @@
 import android.content.Context;
 import android.content.Intent;
 import android.os.Handler;
+import android.os.RemoteException;
 import android.text.TextUtils;
 import android.util.Log;
 
+import com.android.installreferrer.api.InstallReferrerClient;
```

Comment:
won't this trigger a build error if the developer doesn't include the library?

Reply:
A lot of devs will have included this class already but not have added the new dependency

Reply:
This dependency is optional and Install listener class is already built with SDK. The exception will throw at run time only when Java try to load the class, which is already handled in the blow lines

Reply:
Got it. I'd just make sure this is a part of the test cases: Upgrading from an older version of SDK without the gradle inclusion

Reply:
Will ensure this test